src/firecracker/Cargo.toml

```diff
@@ -9,16 +9,16 @@ homepage = "https://firecracker-microvm.github.io/"
 license = "Apache-2.0"
 
 [dependencies]
-event-manager = "0.3.0"
-libc = "0.2.117"
-serde_json = "1.0.78"
-timerfd = "1.3.0"
 
 api_server = { path = "../api_server" }
+event-manager = "0.3.0"
+libc = "0.2.117"
 logger = { path = "../logger" }
 mmds = { path = "../mmds" }
 seccompiler = { path = "../seccompiler" }
-snapshot = { path = "../snapshot"}
+serde_json = "1.0.78"
+snapshot = { path = "../snapshot" }
+timerfd = "1.3.0"
 utils = { path = "../utils" }
 vmm = { path = "../vmm" }
```

Comment:
This intermixes 1st party dependencies with 3p... is there a way for cargo sort to group those accordingly?

Reply:
Just checked there is an option for this (`--grouped`), fixed.